notes/lecture14.tex

```diff
@@ -0,0 +1,273 @@
+\section{Lecture 14: Algorithms using Duality}
+
+The Lagrangian duality theory from the previous lecture can be used to
+design improved optimization algorithms which perform the optimization
+on the dual function. Oftentimes, passing to the dual can simplify
+computation or enable parallelization.
+
+\subsection{Review}
+
+Recall the \emph{primal problem}
+\[
+\min_{x \in \Omega} f(x) : A x = b
```

Comment:
Please write constraint in after `s.t.` and make f(x) align with Ax=b

Reply:
:+1: